libs/utils/src/tabulation.ts

```diff
@@ -310,6 +381,25 @@ export function extractGroupSpecifier(
   };
 }
 
+/**
+ * Convert a {@link Tabulation.GroupMap} to its corresponding {@link Tabulation.GroupList}.
+ * The map format is better for tabulation operations while the list format is easier
+ * preferable for most consumers.
+ */
+export function groupMapToGroupList<T>(
+  groupMap: Tabulation.GroupMap<T>
+): Tabulation.GroupList<T> {
+  const list: Tabulation.GroupList<T> = [];
+  for (const [groupKey, group] of Object.entries(groupMap)) {
+    list.push({
+      ...getGroupSpecifierFromGroupKey(groupKey),
+      // eslint-disable-next-line vx/gts-spread-like-types
```

Comment:
So, this is a fair callout by the linter. It's possible to form a `GroupMap` with with values that are not objects and therefore not spreadable. Looking on the internet, I don't see a good means to define a type which includes any plain object type. Sounds simple, right? If you can find it, I appreciate it. You can't do `T extends {}` because apparently that matches any unknown value. And you can't do `T extends { [key: string]: value }` because that only matches objects with an explicit index signature.

Reply:
Hm, my type-fu isn't up to the challenge, but on the other hand, I've been feeling like the `Tabulation.Group*` types are maybe a little too generic anyway. We only expect to use them with a handful of types related to tallies right? Is there anything stopping us from doing something explicit like this?:

```ts
type TallyData = ElectionResults | CardCounts | CardTally | etc;

type GroupMap<TallyData> = Record<GroupKey, TallyData>;

type GroupOf<TallyData> = TallyData & GroupSpecifier;

// etc...
```

Not 100% sure that solves the eslint warning issue without trying it out, but maybe?

Reply:
The only thing that stops us is that not all the included types are defined at the level of `libs/types` where the `Group*` types are defined.